src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -76,6 +76,24 @@ impl<S: Snapshot> ScannerBuilder<S> {
         self
     }
 
+    /// Set the hint for the minimum commit ts we want to scan.
+    ///
+    /// Default is empty.
+    #[inline]
+    pub fn hint_min_ts(mut self, min_ts: Option<TimeStamp>) -> Self {
+        self.0.hint_min_ts = min_ts;
+        self
+    }
+
+    /// Set the hint for the maximum commit ts we want to scan.
+    ///
+    /// Default is empty.
+    #[inline]
+    pub fn hint_max_ts(mut self, max_ts: Option<TimeStamp>) -> Self {
+        self.0.hint_max_ts = max_ts;
+        self
+    }
+
     /// Build `Scanner` from the current configuration.
     pub fn build(mut self) -> Result<Scanner<S>> {
```

Comment:
I think the `hint_min_ts/hint_max_ts` should be used in `build` method.

Reply:
It's set to the `ScannerConfig` and used by `create_cf_cursor`

Reply:
So is it necessary to include those fields in this builder?